astropy/units/cgs.pyi

```diff
@@ -0,0 +1,656 @@
+# Licensed under a 3-clause BSD style license - see LICENSE.rst
+
+# ruff: noqa: PLC2401
```

Comment:
I would suggest adding the human readable lint id (`non-ascii-name`) to this noqa comment to save future readers from looking it up